### ISSUE TITLE
Document to graph data (not ready for this sprint)

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#55]
## User story
1. As a system
2. I want / need {}
3. So that {}

## Acceptance criteria
* Connect functions that take document to chunks, llm, graph data

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
